Finished landing page

petrykgit opened the pull request:
Working version of the landing page for BOSE.
Demo-link: https://petrykgit.github.io/layout_landing-page/

petrykgit committed:
finished landing page

igorkruz reviewed (changes requested):
Add the demo link to your project, because without it I can’t review it.

petrykgit commented:
> Add the demo link to your project, because without it I can’t review it.

I think I got it. I need to write a demolink in the pull request.

mkyloa reviewed (changes requested):
- add actions to these items

<img width="287" height="332" alt="image" src="https://github.com/user-attachments/assets/baf583ce-e8f3-4faa-bf6b-b58e732d4060" />

- add validation to form
- page should not be reloaded on form submit
- message placeholder has wrong styles
- add hover effects to images

petrykgit committed:
fixed wrong style, added actions

petrykgit commented:
> * add actions to these items...
Added actions - Phone, email, adress.
> * add validation to form
Added validation to form
> * page should not be reloaded on form submit
Made the page doesn't reload on form submit
> * message placeholder has wrong styles
Fixed wrong styles for message placeholder
> * add hover effects to images
Added hover effects to images

781calmstate reviewed (changes requested):
- Message's placeholder has more space on the left then other fields:
<img width="484" height="149" alt="image" src="https://github.com/user-attachments/assets/013f48d7-d6bb-4c7c-8c9f-51e39085b239" />

- The address, phone number and email should have hover effects.
- I can scroll the page while the menu is opened:
<img width="1574" height="594" alt="image" src="https://github.com/user-attachments/assets/0181d335-9486-4fe2-a687-6c4df1b40273" />


### Try to run the `npm test` command, the build has failed and as I can see the issue is some mixins naming

petrykgit committed:
fixed bags

781calmstate reviewed (changes requested):
- **The phone, address and email still have no hover effects.**

- **The page is still scrollable while menu is opened**

<img width="1843" height="700" alt="Image" src="https://github.com/user-attachments/assets/7e963b2c-39cf-4e4a-9381-c8319b37a2aa" />

- **The menu isn't scrollable if the viewport height is smaller than the menu**

<img width="538" height="359" alt="Image" src="https://github.com/user-attachments/assets/99d2e5cb-c5cd-4d5d-bffa-dfd01e984f6d" />

petrykgit committed:
fixed menu mistakes
